modules/402-ingress-nginx/templates/controller/admission.yaml

```diff
@@ -34,7 +34,7 @@ webhooks:
         scope: Namespaced
     failurePolicy: Fail
     sideEffects: None
-    timeoutSeconds: 30
+    timeoutSeconds: 28
```

Comment:
```suggestion
    # set to this value because of DigitalOcean K8s Upgrade requirements (hooks' timeouts should be less than 30 seconds)
    timeoutSeconds: 28
```